yt_dlp/extractor/twitter.py

```diff
@@ -598,7 +598,7 @@ class TwitterIE(TwitterBaseIE):
         # has mp4 formats via mobile API
         'url': 'https://twitter.com/news_al3alm/status/852138619213144067',
         'info_dict': {
-            'id': '852138619213144067',
+            'id': '852077943283097602',
```

Comment:
how does id change?

Reply:
This test had been marked "skip" since before the extractor switched from using the "tweet ID" as video ID to twitter's internal "media ID". The switch was made when Twitter started allowing multiple videos in one tweet. This is also why we make old archive IDs